### ISSUE TITLE
JAWS reads arrow up and arrow down within OverflowMenu with just one item

### ISSUE THREAD

**Issue by ashleylord8** [carbon-design-system/carbon#7452]
## Environment

> Operating system: 
Running Windows 10 VMware (on MacOS Catalina)

> Browser: 
Firefox

> Assistive technology used to verify:
JAWS

## Detailed description

> What version of the Carbon Design System are you using?
"carbon-components": "^10.20.0",
"carbon-components-react": "^7.20.0",

> What did you expect to happen?
If there is only one item within the OverflowMenu, we expect JAWS not to read/say "then navigate with arrow keys" given there's nothing to navigate to.

> What happened instead?
Carbon OverflowMenu makes JAWS say "press space to activate the menu then navigate with arrow keys"... this seems to be a pre-determined aria-label or role that we can't modify if there is only one OverflowMenuItem. And when you open the menu with just one item it says specifically states how to navigate with arrow keys.

## Steps to reproduce the issue

1. OverflowMenu with only ONE OverflowMenuItem
2. With JAWS, open OverflowMenu
3. JAWS will read out "then navigate with arrow keys" even though there's nothing to navigate through with just one OverflowMenuItem.

## Additional information

<img width="367" alt="Screen Shot 2020-12-14 at 9 48 04 AM" src="https://user-images.githubusercontent.com/20712108/102095397-7be5c300-3df1-11eb-86bd-2ca542ac607e.png">

**Comment by dakahn** [comment]
JAWS will read usage instructions for a widget/component even if they're not very helpful (like if there's only one list item). To my knowledge there is no way to circumvent this message and you maybe wouldn't want to since the number of list items could be variable between multiple overflows on the same page and those instructions could be helpful elsewhere. 

The good news is that a habitual or advanced JAWS user may tweak JAWS's verbosity and messages like this would be less frequent.

**Comment by dakahn** [comment]
Closing this up since it's a JAWS feature we can't really control, but happy to reopen if there's something I'm missing. 🏄🏽
